Yoast/javascript
# Rename feature toggle to feature flag

## Summary
* n/a

## Relevant technical choices:

* Cherry picked from https://github.com/Yoast/javascript/pull/246

## Test instructions

This PR can be tested by following these steps:

* Search for `feature-toggle` and make sure that no occurrences of that old name are left.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * Feature flags.

## UI changes

## Quality assurance
